sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/aad/configuration/AzureAuthenticationFilterConfiguration.java

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.autoconfigure.aad.configuration;
+
+import com.azure.spring.cloud.autoconfigure.context.AzureGlobalPropertiesAutoConfiguration;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+@Configuration
+@Import({ AadPropertiesConfiguration.class, AzureGlobalPropertiesAutoConfiguration.class })
+public class AzureAuthenticationFilterConfiguration {
```

Comment:
If it's not a realy test cases class, should we make the class name combine the prefix `Test` ? because they will be confused with classes in `main` source folder,  there's a mistake I fixed in this commit https://github.com/Azure/azure-sdk-for-java/pull/32599/files#diff-25c2618f90491eb545bf00f9d9174590ff8faab4683c2b1499e063447e591f9e
If it's necessary, I will sync this change and update it.

Reply:
Good point, I think it's better we call it `AzureAuthenticationFilterTestConfiguration`? If you agree, I could fix it in my next PR.